src/renderer/src/components/settings-modal/audio-apis.tsx

```diff
@@ -8,13 +8,25 @@ export interface AudioApisProps {
 }
 
 const AudioApis: React.FC<AudioApisProps> = ({ apis, selectedApiId, selectApi }) => {
+  let apiValue = -1;
+
+  // Help the user out a bit by defaulting the audio API to the only valid
+  // entry if no API was selected and the list of available APIs only
+  // has one entry.
+  if (apis.some((api) => api.id === selectedApiId)) {
```

Comment:
You should use find instead of some

Reply:
It's only a "does it contain" check, the code doesn't need the actual api object back, so I used `some`:

> The some() method of array passes the test implemented by the provided function.

Why switch to `find()`?

Reply:
Just for clean code as the api id is unique, using some implies that it may not be.